docker-compose.yaml

```
@@ -12,18 +12,24 @@ services:
       - ./:/go/src
     environment:
       PORT: 8080
+      DB_USER: user
+      DB_PASSWORD: password
+      DB_ADDRESS: docker_db
```

コンテナ名ではなくて，サービス名にしないといけないです！

```suggestion
      DB_ADDRESS: db
```

---

ありがとうございます，修正しました．